dodo.py

```diff
@@ -0,0 +1,29 @@
+def task_build_docker_image():
+    return {
+        "actions": [
+            "docker build -f metaflow/plugins/kfp/tests/Dockerfile -t metaflow-integration-testing:1.0 ."
+        ],
+    }
+
```

Comment:
don't need `doit` in Metaflow, it's another complication, especially since we're only using it for a bash command.